etp-backend/src/main/clj/solita/etp/schema/energiatodistus.clj

```diff
@@ -318,7 +319,8 @@
   (-> schema
       (dissoc :kommentti
               :draft-visible-to-paakayttaja
-              :bypass-validation-limits)
+              :bypass-validation-limits
+              :bypass-validation-limits-reason)
```

Comment:
miksi nämä otetaan pois täältä?

Reply:
Github näyttää tämän väärin. Näitä ei oteta pois EnergiatodistusSave2013 -schemasta vaan ->EnergiatodistusSaveExternal -schemasta.